### PR TITLE
Add cookie banner variant B

### PR DESCRIPTION
**Pull request by BNewing**
We are testing two new cookie banners, and variant B will be hosted on IMB. This PR allows it to pick this new version from Satis, and also makes some changes to the theme to make sure it loads in correctly.

### CODE REVIEW

**Review by brown-a2 — APPROVED** [review]

File: web/app/themes/imb/header.php [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
-			<button class="navbar-toggler" type="button" data-toggle="collapse" data-target=".navbar-collapse"
-					aria-controls=".navbar-collapse" aria-expanded="false" aria-label="Toggle navigation">
-				<span class="icon-bar"></span>
-				<span class="icon-bar"></span>
-				<span class="icon-bar"></span>
-			</button>
-		</div>
+<?php do_action('after_body_open_tag'); ?>
+
+<div class="ccfw-banner-background">
```
Comment (RobLoweMOJ) [inline web/app/themes/imb/header.php:7]: @BNewing the ccfw banner div seems to now wrap the whole page. Just checking if this is correct as it seemed abit odd to have a banner div.

File: web/app/themes/imb/base.php [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
+			<?php require roots_template_path(); ?>
+			</main><!-- /.main -->
+			<?php if ( roots_display_sidebar() ) : ?>
+			<?php endif; ?>
+		</div><!-- /.content -->
   </div><!-- /.wrap -->
 
 	<?php get_footer(); ?>
 
+	</div><!-- /.ccfw-banner-background class set in header -->
```
Comment (RobLoweMOJ) [inline web/app/themes/imb/base.php:21]: Also if this does not need to be wrapping the whole page you could stick the close body and html into the footer.php

File: web/app/themes/imb/header.php (hunk 1 above)
Comment (BNewing) [reply web/app/themes/imb/header.php:7]: The `ccfw-banner-background` div should wrap the rest of the page content, but the cookie banner itself shouldn't. I was aiming for this:

https://www.josephearl.co.uk/post/css-overlay/

It isn't properly working at the moment - I had it working at one point, but must have changed something in between page reloads which is annoying! I made a reference to that on the PR for the banner on the cookie repo but should have put it here - got mixed up with what went where.

**Comment by BNewing** [comment]
I've merged this so we can take a look at it on Monday's demo :) this doesn't want to be pushed any further up than dev though for now!

**Pull request merged**